distributed/nanny.py

```diff
@@ -356,14 +356,17 @@ def start(self):
                 yield self.process.start()
                 if self.status == 'starting':
                     yield gen.with_timeout(timedelta(seconds=5),
```

Comment:
Could we add this timeout to the config file?

Reply:
Pointing this one out as people seem to already be hacking the code to change this anyways as illustrated by [this comment]( https://github.com/dask/distributed/issues/1770#issuecomment-368921388 ) and [this one]( https://github.com/dask/distributed/issues/1770#issuecomment-369013195 ). Even gave it a try myself while doing some debugging earlier. Of course naming is hard, maybe `worker_startup_wait_time`?

Reply:
This is no longer the same timeout.  This is the time for the process to start and send a message back, not the time it takes to connect (which is what is was for the last version)